Gemfile

```diff
@@ -9,3 +9,10 @@ unless RUBY_PLATFORM
   gem 'pry-stack_explorer'
   gem 'pry-rescue'
 end
+
+# Gem a dependency of Capybara, new version of rack 2.0 onwards require ruby 2.2 and above.
+if RUBY_VERSION < "2.2.2"
```

Comment:
On the development question that is correct, but as the Gem file is only being used in development environments no need to specify specific group